src/scss/_themes.scss

```diff
@@ -375,3 +375,25 @@
 		background-size: 35px;
 	}
 }
+
+@mixin oTeaserVideo {
+	.o-teaser__image-container {
+		position: relative;
+
+		&:after {
+			@include oIconsGetIcon('play', oColorsGetPaletteColor('white'), $apply-width-height: false);
+			content: '';
+			position: absolute;
+			bottom: 0;
+			left: 0;
+			background-color: oColorsGetPaletteColor('black');
+			width: 15%;
+			height: 15% * (16 / 9);
```

Comment:
Yeah, that's fixed it. A percentage in padding-top is relative to the _width_, not the _height_. Obviously.

Thanks